examples/nanocoap_server/Makefile

```diff
@@ -35,7 +35,7 @@ USEMODULE += shell_commands
 # Comment this out to disable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-DEVELHELP ?= 1
+DEVELHELP ?= 0
```

Comment:
Nitpick: the comment doesn't make sense if the line says ```?=0```. Feel free to change the line to ```#DEVELHELP ?= 1```